Department-for-Transport-Disruptions/create-disruptions-data
# Approve and reject disruptions

Testing instruction

- Login as staff user and create a disruption or edit a published disruption. Then submit this disruption for approval
- Login as admin or publisher and the disruption should be available to publish or reject
